Kros.Utils/src/Kros.Utils/Properties/Resources.resx

```diff
@@ -117,79 +117,94 @@
+    <value>Argument value must be {0}. Current value is {1}.</value>
   </data>
   <data name="Check_GreaterOrEqualThan" xml:space="preserve">
-    <value>Hodnota parametra musí byť väčšia alebo rovná {0}, ale skutočná hodnota je {1}.</value>
+    <value>Argument value must be greater or equal than {0}. Current value is {1}.</value>
   </data>
   <data name="Check_GreaterThan" xml:space="preserve">
-    <value>Hodnota parametra musí byť väčšia ako {0}, ale skutočná hodnota je {1}.</value>
+    <value>Argument value must be greater than {0}. Current value is {1}.</value>
   </data>
   <data name="Check_IsInListDefaultMessage" xml:space="preserve">
-    <value>Hodnota "{0}" musí byť zo zoznamu hodnôt "{1}".</value>
+    <value>The value "{0}" is invalid. It must be one of the allowed values: {1}</value>
   </data>
   <data name="Check_IsNotInListDefaultMessage" xml:space="preserve">
-    <value>Hodnota "{0}" nesmie byť zo zoznamu hodnôt "{1}".</value>
+    <value>The value "{0}" is invalid. It cannot be one of disallowed values: {1}</value>
   </data>
   <data name="Check_IsNotOfType" xml:space="preserve">
-    <value>Parameter nemôže byť typu {0}.</value>
+    <value>Argument cannot be of type {0}.</value>
   </data>
   <data name="Check_IsOfType" xml:space="preserve">
-    <value>Parameter nie je požadovaného typu. Požadovaný typ: {0}. Typ parametra: {1}.</value>
+    <value>Argument is not of required type {0}. Argument's type is {1}.</value>
   </data>
   <data name="Check_LessOrEqualThan" xml:space="preserve">
-    <value>Hodnota parametra musí byť menšia alebo rovná {0}, ale skutočná hodnota je {1}.</value>
+    <value>Argument value must be less or equal than {0}. Current value is {1}.</value>
   </data>
   <data name="Check_LessThan" xml:space="preserve">
-    <value>Hodnota parametra musí byť menšia ako {0}, ale skutočná hodnota je {1}.</value>
+    <value>Argument value must be less than {0}. Current value is {1}.</value>
   </data>
   <data name="Check_NotEmptyGuid" xml:space="preserve">
-    <value>Prázdny GUID nie je povolený.</value>
+    <value>Empty GUID is not allowed.</value>
   </data>
   <data name="Check_NotEqual" xml:space="preserve">
-    <value>Hodnota parametrov nesmie byť rovnaká {0}.</value>
+    <value>Argument value {0} is not allowed.</value>
   </data>
   <data name="Check_NotNull" xml:space="preserve">
-    <value>Hodnota nemôže byť null.</value>
+    <value>The value cannot be null.</value>
   </data>
   <data name="Check_StringNotNullOrEmpty" xml:space="preserve">
-    <value>Reťazec nemôže byť null, alebo prázdny.</value>
+    <value>The value cannot be null, neither empty string.</value>
   </data>
   <data name="Check_StringNotNullOrWhiteSpace" xml:space="preserve">
-    <value>Reťazec nemôže byť null, prázdny, ani zložený iba z bielych znakov.</value>
+    <value>The value cannot be null, nither empty string, nor string containing only whitespace characters.</value>
```

Comment:
- nither -> neither
- nor -> or?

Reply:
Will correct "nither". "nor" is good. ;)